android/config.gradle

```diff
@@ -53,4 +53,4 @@ ext.androidCompileSdkVersion = 28
 ext.androidBuildToolsVersion = '28.0.3'
 ext.androidMinSdkVersion = 19
 ext.androidTargetSdkVersion = 28
-ext.androidGradlePluginVersion = '3.2.1'
\ No newline at end of file
+ext.androidGradlePluginVersion = '3.2.1'
```

Comment:
In the framework, we're actually already using version 3.3.2. Is there a reason for flutter to not use the latest stable version of the plugin?

Reply:
`3.2.1` is the Gradle plugin version used by Flutter currently. This can be verified by typing `flutter create exampleApp` and the newly created example will use 3.2.1. 
I've tried to update to `3.3.2` and it actually works, but I would play safe and use the same version that the Flutter team uses.